h2o-core/src/main/java/hex/Model.java

```diff
@@ -5,6 +5,8 @@
 import hex.genmodel.easy.RowData;
 import hex.genmodel.easy.exception.PredictException;
 import hex.genmodel.easy.prediction.*;
+import org.apache.commons.compress.archivers.zip.ZipArchiveEntry;
```

Comment:
I don't remember exactly :) I started with `java.util.zip.*`, but then had to switch to org.apache's version because standard java zip library didn't support the functionality that I needed. Of course, since then it was all refactored many times, so maybe I don't need that functionality anymore...

One thing that org.apache version does better is that it's very easy to switch to a different archive format (for example TAR).